Sources/Common/Core/LookupTable/index.js

```diff
@@ -272,6 +272,11 @@ function vtkLookupTable(publicAPI, model) {
     // Add "special" colors (NaN, below range, above range) to table here.
     const { numberOfColors } = model;
 
+    // Resize the array to hold three additional colors
+    model.table.push(0, 0, 0, 0);
+    model.table.push(0, 0, 0, 0);
+    model.table.push(0, 0, 0, 0);
+
```

Comment:
Might not be needed since doing that `array[array.length] = 5` is valid and will technically extend the size of the array. ;-)

Reply:
Ah ok. I'll remove it then.